Formalisation(shacl)/Core/PiecesShape/DataService.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dash: <http://datashapes.org/dash#> .
+
+
+:DataServiceShape a sh:NodeShape ;
+  sh:targetClass dcat:DataService ;
+  sh:property [
+    sh:path dcat:endpointURL ;
+    sh:nodeKind sh:IRI ;
+    sh:minCount 1 ;
+    sh:name "Endpoint URL" ;
+    dash:editor dash:URIEditor ;
+    dash:viewer dash:URIViewer ;
+  ],
+  [
+    sh:path dct:title ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LiteralViewer ;
+    sh:minCount 1 ;
+    sh:uniqueLang true ;
+  ],
+  [
+    sh:path dct:endpointDescription ;
+    sh:nodeKind sh:Literal ;
+    dash:editor dash:TextFieldEditor ;
+    dash:viewer dash:LiteralViewer ;
+    sh:uniqueLang true ;
```

Comment:
`dcat:endpointDescription` should point to a `rdfs:Resource` according to the spec, so that means an IRI in our case. Generally if we can get an OpenAPI description that would be great.

```suggestion
    sh:path dcat:endpointDescription ;
    sh:nodeKind sh:IRI ;
    dash:viewer dash:URIViewer ;
    dash:editor dash:URIEditor ;
```

Reply:
according to the diagram, it is not rdfs:Resource. please see below:
![image](https://github.com/Health-RI/health-ri-metadata/assets/2759916/56d8975c-f995-4467-a89c-a6c4b9416385)

Reply:
@xiaofengleo  seems diagram is wrong, see https://www.w3.org/TR/vocab-dcat-3/#Property:data_service_endpoint_description and https://semiceu.github.io/DCAT-AP/releases/3.0.0/#DataService

Reply:
Might be a conscious choice of the data team, but it's clearly conflicting with the core spec. No reason for us to deviate from that. I'll make an issue for it.

Reply:
@kburger do we wait till the diagram being updated by data team or revise the schacl for now?